.github/workflows/release-cli.yml

```diff
@@ -531,10 +531,13 @@ jobs:
           curl "${BASE_URL}/msi/${MSI_FILE}" --output install.msi
 
       - name: Install on Windows from MSI
-        shell: bash
+        timeout-minutes: 2
         run: |
-          msiexec /i install.msi /quiet /qn /l*vx install_log.txt
-          [ ! -d "C:\Program Files\Keboola CLI" ] && exit 1
+          msiexec /i install.msi /quiet /qn /le install_log.txt
+          if (!(Test-Path "C:\Program Files\Keboola CLI")) {
+            cat install_log.txt
+            Exit 1
+          }
```

Comment:
Ked to timeoutuje, tak sa ti log nevypise, mozno by to bolo lepsie dat ako dalsi step s `always()`.

Reply:
Jo jo, to vím, nečekal jsem že to vyřeší ten timeout. Ahá, ale když to přidám jako samostatný krok, tak by to mohlo pomoct. 👍 (Pokud se ten log teda vůbec vytvoří při tom timeoutu. Ale nic tím nezkazím.)